src/main/java/lotto/domain/lotto/LottoTicket.java

```diff
@@ -1,26 +1,19 @@
-package lotto.domain;
+package lotto.domain.lotto;
 
 import java.util.*;
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
 
 public class LottoTicket {
 
-    private final static int LOTTO_MAX_NUMBER = 45;
-    private final static int LOTTO_MIN_NUMBER = 1;
-    private final static int LOTTO_TICKET_SIZE = 6;
-
     private final static int LOTTO_MATCH  = 1;
     private final static int LOTTO_NOT_MATCH  = 0;
-
-    private final static List<LottoNumber> lottoAllNumbers = IntStream.rangeClosed(LOTTO_MIN_NUMBER, LOTTO_MAX_NUMBER)
-            .mapToObj(LottoNumber::create)
-            .collect(Collectors.toList());
+    private final static int LOTTO_TICKET_SIZE = 6;
 
     private List<LottoNumber> lottoNumbers = new ArrayList<>();
```

Comment:
일급컬렉션의 특징 중 하나는 불변성을 가지고 있기 때문에 fianl 키워드를 붙이면 좋을 것 같아요 :)
```suggestion
    private final List<LottoNumber> lottoNumbers;
```